packages/auth/README.md

```diff
@@ -118,11 +126,11 @@ The following values are available from the `useAuth` hook:
 * async `getToken()`: returns a jwt
 * `client`: Access the instance of the client which you passed into `AuthProvider`
 * `authenticated`: used to determine if the current user has authenticated
-* `loading`: The auth state is restored asynchronously when the user visits the site for teh first time, use this to determine if you have the correct state
+* `loading`: The auth state is restored asynchronously when the user visits the site for the first time, use this to determine if you have the correct state
 
-## Usage in Redwood
+## Usage in RedwoodJS
```

Comment:
Should this be "Redwood?"

Reply:
yeah, honestly idk -- changing back for continuity here.

I just made the natural change when I saw it in the header. I then re-read and questioned myself but decided to leave it.